packages/react-native/React/Views/RCTConvert+Transform.h

```diff
@@ -9,6 +9,6 @@
 
 @interface RCTConvert (Transform)
 
-+ (CATransform3D)CATransform3D:(id)json;
++ (CATransform3D)CATransform3D:(id)json viewWidth: (CGFloat) viewWidth viewHeight: (CGFloat) viewHeight transformOrigin: (NSString*) transformOrigin;
```

Comment:
For backwords compatibility we should keep the old method signature around too (but have it just call the new method with 0 widht/height and nil origin)